docs/source/quickstart.rst

```diff
@@ -91,187 +113,206 @@ as follows (this example is also included in ``quickstart/mlflow_tracking.py``):
-runs using ``mlflow ui``.
+    .. code-block:: shell
 
-.. note::
-    By default ``mlflow run`` installs all dependencies using `virtualenv <https://virtualenv.pypa.io/en/latest//>`_.
-    To run a project without using ``virtualenv``, you can provide the ``--env-manager=local`` option to
-    ``mlflow run``. In this case, you must ensure that the necessary dependencies are already installed
-    in your Python environment.
+        mlflow server
 
-For more information, see :doc:`projects`.
+For instance, if you've run the above command on a machine with IP address **192.168.0.1** and port **5000**, you can add tracking data to it either by:
 
-Saving and Serving Models
--------------------------
+.. code-section::
 
-MLflow includes a generic ``MLmodel`` format for saving *models* from a variety of tools in diverse
-*flavors*. For example, many models can be served as Python functions, so an ``MLmodel`` file can
-declare how each model should be interpreted as a Python function in order to let various tools
-serve it. MLflow also includes tools for running such models locally and exporting them to Docker
-containers or commercial serving platforms.
+    .. code-block:: python
 
-To illustrate this functionality, the ``mlflow.sklearn`` package can log scikit-learn models as
-MLflow artifacts and then load them again for serving. There is an example training application in
-``sklearn_logistic_regression/train.py`` that you can run as follows:
+        mlflow.set_tracking_uri("http://192.168.0.1:5000")
+        mlflow.autolog()  # Or other tracking functions
 
-.. code-block:: bash
+Or, on your development machine, you can set the ``MLFLOW_TRACKING_URI`` environment variable to the URL of that server:
 
-    python sklearn_logistic_regression/train.py
+.. code-section::
 
-When you run the example, it outputs an MLflow run ID for that experiment. If you look at
-``mlflow ui``, you will also see that the run saved a ``model`` folder containing an ``MLmodel``
-description file and a pickled scikit-learn model. You can pass the run ID and the path of the model
-within the artifacts directory (here "model") to various tools. For example, MLflow includes a
-simple REST server for python-based models:
+    .. code-block:: shell
 
-.. code-block:: bash
+        export MLFLOW_TRACKING_URI=http://192.168.0.1:5000
 
-    mlflow models serve -m runs:/<RUN_ID>/model
+Now, when you run your code, it will send tracking data to the tracking server. You can view the tracking data by navigating to the URI with a browser.
 
-.. note::
+There are many options available for the tracking backend. For more details, see :ref:`tracking_server`.
 
-    By default the server runs on port 5000. If that port is already in use, use the `--port` option to
-    specify a different port. For example: ``mlflow models serve -m runs:/<RUN_ID>/model --port 1234``
+Use MLflow with a Databricks workspace
```

Comment:
I think we should make this a subheader of "Share MLflow Runs and experiments"

Reply:
~~If we do this, it will drop off the TOC. ~~ DONE